web/src/global-styles/buttons.scss

```
@@ -26,7 +26,7 @@
         @extend .icon-inline;
     }
 
-    &:focus:not(:disabled):not(.disabled) {
+    &.focus-visible:not(:disabled):not(.disabled) {
```

Should this be `:focus-visible` too thanks to postcss?
```suggestion
    &:focus-visible:not(:disabled):not(.disabled) {
```